sdk/metricsadvisor/Azure.AI.MetricsAdvisor/perf/MetricsAdvisorTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Test.Perf;
+
+namespace Azure.AI.MetricsAdvisor
```

Comment:
You can add a suffix such as `.Perf` or `.Perf.Scenarios` in namespace of the perf test classes.